### PR TITLE
Extend supported exp.date formats.

### PR DESCRIPTION
**Pull request by dmytrokhl**
## Description of changes
Support exp.date in format where year-first: `yy/mm` or `yyyy/mm`

Related to https://github.com/verygoodsecurity/vgs-collect-ios/issues/259.

### CODE REVIEW

File: Tests/FrameworkTests/ConvertorsTests/ExpDateConvertorTests.swift [hunk 1]
```diff
@@ -119,4 +119,179 @@ class ExpDateConvertorTests: VGSCollectBaseTestCase {
 			}
     }
   }
+  
+  func testConvertExpDate5() {
+    let config = VGSExpDateConfiguration(collector: collector, fieldName: "textField")
+
```
Comment (EugeneIOs) [inline Tests/FrameworkTests/ConvertorsTests/ExpDateConvertorTests.swift:125]: add test to scanners mapping

File: Sources/VGSCardIOCollector/CardIODataMapper/VGSCardIODataMapUtils.swift [hunk 2]
```diff
@@ -35,6 +35,10 @@ internal final class VGSCardIODataMapUtils {
 			return mapYear(data.year)
 		case .expirationYearLong:
 			return mapYearLong(data.year)
+    case .expirationDateShortYearThenMonth:
+      return mapExpirationDateWithShortYearFirst(data.month, scannedExpYear: data.year)
+    case .expirationDateLongYearThenMonth:
+      return mapLongExpirationDateWithLongYearFirst(data.month, scannedExpYear: data.year)
```
Comment (EugeneIOs) [inline Sources/VGSCardIOCollector/CardIODataMapper/VGSCardIODataMapUtils.swift:41]: check date picker case

**Comment by japborst** [comment]
Hey @dmytrokhl, we'd love to use this feature. Do you have an idea when this would land?
Alternative is that we'd implement our own expiration field logic...

**Comment by dmytrokhl** [comment]
Hi @japborst, we plan to release it this week.

File: Sources/VGSCollectSDK/Utils/Helpers/Mappers/VGSExpirationDateTextFieldUtils.swift [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+//  Created by Dima on 21.04.2021.
+//  Copyright © 2021 VGS. All rights reserved.
+//
+
+import Foundation
+
+internal final class VGSExpirationDateTextFieldUtils {
+  
+  /// Map the date components to expected VGSCardExpDateFormat
+  internal static func mapDatePickerExpirationDataForFieldFormat(_ format: VGSCardExpDateFormat, month: Int, year: Int) -> String {
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/Utils/Helpers/Mappers/VGSExpirationDateTextFieldUtils.swift:14]: add unit tests for this function

**Review by EugeneIOs — APPROVED** [review]

**Pull request merged**